### DISCUSSION TITLE
Can HTML be embedded in pixijs

### DISCUSSION THREAD

**Discussion by a6965921** [pixijs/pixijs#7410] [Q&A]
Can HTML be embedded in pixijs

It's written in HTML mixed with jspixi

**Comment by bigtimebuddy** [comment]
You can use SVG's [foreignObject](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/foreignObject) to render HTML to SVG. Then it can then be used by Pixi to create a texture.
